packages/desktop/electron/electronApi.js

```diff
@@ -314,6 +315,10 @@ const ElectronApi = {
                 }
             })
     },
+
+    isFeatureFlagEnabled: (keyPath) => {
+        return keyPath?.split('.').reduce((prev, cur) => prev && prev[cur], features)?.enabled ?? false
+    },
```

Comment:
To keep the consistency with the other functions, this function shouldn't be written as an arrow function.